Singlestore/api integration (#2045)
DataDog/integrations-extras: singlestoredb_cloud/README.md

4. Select **Create**. This action takes you to the Datadog sign-in page. After signing in to Datadog, proceed to the next step.
5. On the **Authorize access** screen, select the **Authorize** button. Upon successful authorization, you are taken to the **Integration** page on the Cloud Portal. 

You can now monitor your SingleStoreDB Cloud databases using Datadog.

### Uninstall the Datadog Integration

To uninstall/deactivate the Datadog integration,

1. **Uninstall the SingleStore integration on Datadog**: On the Datadog dashboard, select **Integrations > SingleStore > Configure > Uninstall Integration**. Once this integration is uninstalled, all previous authorizations are revoked.
2. **Remove the Datadog integration on the Cloud Portal**: On the Cloud Portal, go to **Monitoring > Integration**. Select **Delete** for each Datadog configuration you want to remove. 

Additionally, remove all the API keys associated with this integration.

To stop monitoring a specific workspace group (and not uninstall the integration), select **Delete** (**Cloud Portal > Monitoring > Integration**) to remove the Datadog configuration for this workspace group.

## Data Collected

> ### Metrics
> 
> See [metadata.csv][2] for a list of metrics provided by this integration.
> 

Reviewer: ```suggestion
### Metrics

See [metadata.csv][2] for a list of metrics provided by this integration.

### Service Checks

SingleStoreDB Cloud does not include any service checks. 

### Events

SingleStoreDB Cloud does not include any events. 

```

## Overview

SingleStoreDB Cloud is a distributed, relational database with optimized speed and scalability to support data-intensive and real-time applications. Using the Datadog integration, you can monitor your SingleStoreDB Cloud databases. To integrate Datadog with SingleStoreDB Cloud, install the Datadog SingleStore integration and then configure it in the [Cloud Portal][1]. 

> 

Author: Updated overview section to include the value "Using this Integration, you can monitor overall health and performance of a SinglestoreDB Cloud Workspace Group/Cluster"

> 

Reviewer 2: Hi @tdasarathan , more specifically I think it would be useful if you mentioned:
- the types of health and performance data offered
- what this data looks like (will it be in metrics, logs, etc?)

> 

Author: Updated, please review

## Setup

Start the configuration process from Datadog and complete the configuration in the SingleStore Cloud Portal:

1. [Install the SingleStoreDB Cloud Integration in Datadog](#install-the-singlestoredb-cloud-integration-in-datadog)
2. [Configure the Datadog Integration in the Cloud Portal](#configure-the-datadog-integration-in-the-cloud-portal)

> 

Author: deleted this section

> ### Install the SingleStoreDB Cloud Integration in Datadog

Reviewer: ```suggestion
### Install the SingleStoreDB Cloud integration in Datadog
```

### Configure the Datadog integration in the Cloud Portal

To connect your SingleStoreDB Cloud workspace group with Datadog:

1. Sign in to the Cloud Portal. Upon signing in, you are taken to the **Integration** page. You can also select **Monitoring > Integration** on the left navigation pane to access this page.
2. From the list of available integrations, select **+ Integration** for Datadog.
3. On the **Create Datadog Integration** dialog, from the **Workspace Group** list, select your workspace group.
4. Select **Create**. This action takes you to the Datadog sign-in page. After signing in to Datadog, proceed to the next step.
5. On the **Authorize access** screen, select the **Authorize** button. Upon successful authorization, you are taken to the **Integration** page on the Cloud Portal. 

You can now monitor your SingleStoreDB Cloud databases using Datadog.

> ### Uninstall the Datadog Integration

Reviewer: ```suggestion
### Uninstall the Datadog integration
```

Follow these steps to uninstall the Datadog integration:

1. **Uninstall the SingleStoreDB Cloud integration on Datadog**: On the Datadog dashboard, select **Integrations > SingleStoreDB Cloud > Configure > Uninstall Integration**. Once this integration is uninstalled, all previous authorizations are revoked.
2. **Remove the Datadog integration on the Cloud Portal**: On the Cloud Portal, go to **Monitoring > Integration**. Select **Delete** for each Datadog configuration you want to remove. 

Additionally, remove all the API keys associated with this integration.

> To stop monitoring a specific workspace group (and not uninstall the integration), select **Delete** (**Cloud Portal > Monitoring > Integration**) to remove the Datadog configuration for this workspace group.

Reviewer: ```suggestion
To stop monitoring a specific workspace group (and retain the integration), select **Delete** (**Cloud Portal > Monitoring > Integration**) to remove the Datadog configuration for this workspace group.
```

## Overview

> SingleStoreDB Cloud is a distributed, relational database with optimized speed and scalability to support data-intensive and real-time applications. Using this Integration, you can monitor overall health and performance of a SinglestoreDB Cloud Workspace Group/Cluster. To integrate Datadog with SingleStoreDB Cloud, install the Datadog SingleStore integration and then configure it in the [Cloud Portal][1]. 

Reviewer 2: ```suggestion
SingleStoreDB Cloud is a distributed, relational database with optimized speed and scalability to support data-intensive and real-time applications. Using this integration, you can monitor the overall health and performance of a SinglestoreDB Cloud Workspace Group/Cluster. To integrate Datadog with SingleStoreDB Cloud, install the Datadog SingleStore integration and then configure it in the [Cloud Portal][1]. 
```
